### PR TITLE
sysadmins: Document CentOS 6 Python 2.7 and Ice 3.5 installation

### PR DESCRIPTION
**Pull request by ghost**
This describes the manual steps to set up Python and Ice, and also how to configure the environment to start up the server.

I've tested this on a bare CentOS 6 install, and if you repeat the instructions you should get something similar to below.  You should also be able to connect with insight, upload and view images etc.

/cc @joshmoore @chris-allan @stick 

```
[rleigh@localhost OMERO.server-5.1.4-ice35-b55]$ bin/omero admin start
Creating /opt/omero/OMERO.server-5.1.4-ice35-b55/var/master
Creating /opt/omero/OMERO.server-5.1.4-ice35-b55/var/registry
No descriptor given. Using etc/grid/default.xml
Waiting on startup. Use CTRL-C to exit
[rleigh@localhost OMERO.server-5.1.4-ice35-b55]$ bin/omero admin diagnostics

================================================================================
OMERO Diagnostics 5.1.4-ice35-b55
================================================================================

Commands:   java -version                  1.8.0     (/usr/bin/java)
Commands:   python -V                      2.7.5     (/opt/rh/python27/root/usr/bin/python -- 2 others)
Commands:   icegridnode --version          3.5.1     (/opt/Ice-3.5.1/bin/icegridnode)
Commands:   icegridadmin --version         3.5.1     (/opt/Ice-3.5.1/bin/icegridadmin)
Commands:   psql --version                 9.4.5     (/usr/bin/psql)

Server:     icegridnode                    running
Server:     Blitz-0                        active (pid = 79981, enabled)
Server:     DropBox                        active (pid = 80000, enabled)
Server:     FileServer                     active (pid = 79998, enabled)
Server:     Indexer-0                      active (pid = 79996, enabled)
Server:     MonitorServer                  active (pid = 80011, enabled)
Server:     OMERO.Glacier2                 active (pid = 80005, enabled)
Server:     OMERO.IceStorm                 active (pid = 80014, enabled)
Server:     PixelData-0                    active (pid = 80013, enabled)
Server:     Processor-0                    active (pid = 80027, enabled)
Server:     Tables-0                       active (pid = 80042, enabled)
Server:     TestDropBox                    inactive (enabled)

Log dir:    /opt/omero/OMERO.server-5.1.4-ice35-b55/var/log exists

Log files:  Blitz-0.log                    69.0 KB      
Log files:  DropBox.log                    1.0 KB       
Log files:  FileServer.log                 0.0 KB       
Log files:  Indexer-0.log                  2.0 KB        errors=0    warnings=1   
Log files:  MonitorServer.log              1.0 KB       
Log files:  OMEROweb.log                   n/a
Log files:  PixelData-0.log                2.0 KB       
Log files:  Processor-0.log                2.0 KB       
Log files:  Tables-0.log                   0.0 KB       
Log files:  TestDropBox.log                n/a
Log files:  master.err                     0.0 KB       
Log files:  master.out                     0.0 KB       
Log files:  Total size                     0.08 MB


Environment:OMERO_HOME=(unset)             
Environment:OMERO_NODE=(unset)             
Environment:OMERO_MASTER=(unset)           
Environment:OMERO_USERDIR=(unset)          
Environment:OMERO_TMPDIR=(unset)           
Environment:PATH=/opt/Ice-3.5.1/bin:/opt/rh/python27/root/usr/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin:/sbin:/home/rleigh/bin 
Environment:PYTHONPATH=/opt/Ice-3.5.1/python: 
Environment:ICE_HOME=(unset)               
Environment:LD_LIBRARY_PATH=/opt/Ice-3.5.1/lib64:/opt/Ice-3.5.1/lib:/opt/rh/python27/root/usr/lib64 
Environment:DYLD_LIBRARY_PATH=(unset)      

OMERO SSL port:4064                           
OMERO TCP port:4063                           
OMERO data dir:'/opt/omero/data/'             Exists? True  Is writable? True
OMERO temp dir:'/home/rleigh/omero/tmp'       Exists? True  Is writable? True   (Size: 20384)

JVM settings: Blitz                         -Xmx600m -XX:MaxPermSize=512m -XX:+IgnoreUnrecognizedVMOptions
JVM settings: Indexer                       -Xmx400m -XX:MaxPermSize=512m -XX:+IgnoreUnrecognizedVMOptions
JVM settings: Pixeldata                     -Xmx600m -XX:MaxPermSize=512m -XX:+IgnoreUnrecognizedVMOptions
JVM settings: Repository                    -Xmx400m -XX:MaxPermSize=512m -XX:+IgnoreUnrecognizedVMOptions

OMERO.web status... WARNING: FastCGI support is deprecated and will be removed in OMERO 5.2. Install Gunicorn and update config.
[NOT STARTED]
```

### CODE REVIEW

File: omero/sysadmins/unix/server-centos6.txt [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+=============================================
+
+Running OMERO on CentOS 6 has a number of special requirements which
+deviate from the standard installation instructions.  The instructions
+below will set up Python 2.7 and Ice 3.5 on CentOS 6.
+
+Python 2.7
+----------
+
+CentOS 6 provides Python 2.6.  However, OMERO.web requires Python 2.7
```
Comment (joshmoore) [inline omero/sysadmins/unix/server-centos6.txt]: re: "requires" -- terminology here should match gh-1293

Comment (joshmoore) [inline omero/sysadmins/unix/server-centos6.txt]: Or perhaps better put: there are 2 ways to read this. Perhaps clarify that "in order to use Django 1.8, Python 2.7 is required".

**Comment by joshmoore** [comment]
Otherwise, makes sense. Leaving you and @manics to decide the zip/rpm/job work which will test this. (There's some danger of those getting out of sync. If that can be fixed via an `@include`, then :+1:)

File: omero/sysadmins/unix/server-centos6.txt [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+in order to use django 1.8.  While django 1.6 may be used with Python
+2.6, this version of django no longer has security support.  In
+consequence, it's necessary to upgrade to Python 2.7 in order to
```
Comment (mtbc) [inline omero/sysadmins/unix/server-centos6.txt]: contraction

File: omero/sysadmins/unix/server-centos6.txt (hunk 1 above)
Comment (rleigh-dundee) [inline omero/sysadmins/unix/server-centos6.txt]: The sentence already says this though doesn't it?  "... requires Python 2.7 in order to use django 1.8".  I can rearrange the sentence but the meaning is already exactly this.

**Comment by rleigh-dundee** [comment]
Is this good to merge or do you want additional changes?

**Comment by hflynn** [comment]
This still needs signing off @joshmoore

**Comment by jburel** [comment]
@hflynn: listed for tomorrow standup

**Comment by manics** [comment]
> Otherwise, makes sense. Leaving you and @manics to decide the zip/rpm/job work which will test this. (There's some danger of those getting out of sync. If that can be fixed via an @include, then :+1:)

e.g. omero-install? That's somewhere on my todo list (and a downloadable RPM for ice would be nice)...

A general comment, there's no top-level overview to tell people where to start reading. These are both pages of links:
https://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/
https://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/unix/index.html

File: omero/sysadmins/unix/server-centos6.txt [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+Development packages
+--------------------
+
+The following sections require a number of libraries and a compiler.
+Run the following commands to install them::
+
+    yum groupinstall "Development Tools"
+    yum install epel-release
+    # Ice dependencies
+    yum install expat-devel libbz2-devel openssl-devel zlib-devel
```
Comment (manics) [inline omero/sysadmins/unix/server-centos6.txt]: `libbz2-devel` -> `bzip2-devel`

**Comment by rleigh-dundee** [comment]
@jburel Fix pushed.

**Comment by jburel** [comment]
Thanks

**Comment by manics** [comment]
Looks good to me, here's the scripts I used for testing: https://github.com/manics/omero-install/compare/wsgi-release...manics:wsgi-release-c6-py27?expand=1

**Comment by jburel** [comment]
Thanks @rleigh-dundee. Merging

**Pull request merged**